src/plugins/dm.c

```diff
@@ -516,6 +516,29 @@ static void find_raid_sets_for_dev (const gchar *name, const gchar *uuid, gint m
         }
     }
 }
+
+static gboolean get_raid_set_status (struct raid_set *rs, GError **error) {
+    struct dm_task *dmt = NULL;
+    struct dm_info info;
+    gboolean ret;
+
+    dmt = dm_task_create (DM_DEVICE_STATUS);
+    if (!dmt) {
+        g_set_error (error, BD_DM_ERROR, BD_DM_ERROR_TASK,
+                     "Failed to create DM task");
+        return FALSE;
+    }
+
+    if (!dm_task_set_name (dmt, rs->name) || !dm_task_run (dmt) || !dm_task_get_info (dmt, &info)) {
```

Comment:
I'm not sure which of these can fail because the DM device doesn't exist and which means something is really broken (and libdevmapper "documentation" is not very helpful here) so I've decided to just ignore everything here. Worst case scenario is the RAID set change will fail so same behaviour we have now for some weird/unsupported cases.

Reply:
It's little bit too long for the comment, but I've adjusted the comment a little to contain more information.